libutils/file_lib.c

```diff
@@ -466,8 +466,8 @@ Seq *ListDir(const char *dir, const char *extension)
 
 mode_t SetUmask(mode_t new_mask)
 {
-    const mode_t old_mask = umask(0077);
-    Log(LOG_LEVEL_DEBUG, "Set umask to 0077, was %o", old_mask);
```

Comment:
Why not `777` ?

Reply:
Because it's a mask. `007` always wears a mask to be sneaky.